.github/workflows/tests_on_push.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  push:
+    branches: '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checkout code
+    - uses: actions/checkout@CI
```

Comment:
It doesn't work with `CI`

the branch name is the name of the checkout utility not of your branch. Just revert to master.